App/Models/TextileTypes.ts

```diff
@@ -125,7 +125,7 @@ export interface Photo {
   username?: string
   metadata?: PhotoMetadata
   comments: UserComment[]
-  Likes: UserLike[]
+  likes: UserLike[]
```

Comment:
No idea how this was working, must have been in JS code where the TS compiler wasn't checking. Or on @andrewxhill's computer where TS doesn't even check.